src/software/backend/input/network/filter/ball_filter_test.cpp

```diff
@@ -545,5 +545,5 @@ TEST_F(BallFilterTest,
     // Check the lines are pointing in the same direction
     EXPECT_LT(x_vs_y_regression.regression_line.toNormalUnitVector().cross(
                   y_vs_x_regression.regression_line.toNormalUnitVector()),
-              GeomConstants::FIXED_EPSILON);
+              ::FIXED_EPSILON);
```

Comment:
```suggestion
              FIXED_EPSILON);
```

Reply:
Whoops missed this in the rebase. I think we should just stick with one since it's the same with and without the `::` prefix.